src/components/syntax-highlighter/syntax-highlighter.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import Highlighter from "react-syntax-highlighter";
+
+const vegaJsonTheme = {
+  hljs: {
+    fontSize: "1rem",
+    fontFamily: "Consolas, Monaco, 'Andale Mono', 'Ubuntu Mono', monospace",
+    display: "block",
+    overflowX: "auto",
+    padding: "1em",
+    color: "#26ff8a",
+    background: "#2C2C2C",
+    border: "1px solid #696969",
+  },
+  "hljs-literal": {
+    color: "#ff2d5e",
+  },
+  "hljs-number": {
+    color: "#ff7a1a",
+  },
+  "hljs-string": {
+    color: "#48aff0",
+  },
+};
+
+/**
+ * Remove nested __typename from response as it's useless
+ */
+const removeTypename = (data: object): any => {
```

Comment:
You could reach for lodash.omit if ya felt like it, although omitDeep isn't a thing so you'd still need to handle that

Reply:
Actually I do not need this any more so I can just remove it